src/App.vue

```diff
@@ -58,6 +59,11 @@
 				@perso-bills-created="onPersoBillsCreated"
 				@duplicate-bill="onDuplicateBill"
 				@repeat-bill-now="onRepeatBillNow" />
+			<ProjectList
+				v-else-if="mode === 'move'"
+				:bill="currentBill"
+				:project-id="currentProjectId"
+				@item-moved="onBillMoved" />
```

Comment:
A Modal (from `@nextcloud/vue`) would fit perfectly for this. Few information, the action is required right now, it's easier to understand what happens after clicking "move" etc...

So, <ProjectList> could be a modal or it could just be contained in a modal (I prefer the second one):
``` vue
<Modal v-if="showMoveModal"
	size="small"
	@close="showMoveModal = false">
	<ProjectList
		:bill="billToMove"
		:project-id="currentProjectId"
		@item-moved="onBillMoved" />
</Modal>
```
This way you don't use the `mode` data prop but add an independent `showMoveModal` one.
Also, I don't think that's necessary to select the bill we want to move. Just use a `billToMove` data prop which is set in `onMoveBillClicked`. See next comment.